packages/react/src/components/Card/Card.jsx

```diff
@@ -215,6 +215,7 @@ export const defaultProps = {
     extra: false,
   },
   renderExpandIcon: undefined,
+  renderInPortal: false,
```

Comment:
Should this default be true? I think the DateTimePickerV2 was built specifically to render in a portal for the Cards. This is most likely why the story is broken